karma.conf.js

```diff
@@ -0,0 +1,106 @@
+module.exports = function(config) {
+
+  'use strict';
```

Comment:
Oh, I see the problem. By changing `index.js` to `*.js` in the makefile we're linting __karma.conf.js__ with the same options as __index.js__. Instead, let's add a separate command:

```make
	$(ESLINT) \
	  --env node \
	  -- karma.conf.js
```